GameOfLife/Calculations/GameFieldSize.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    class GameFieldSize
+    {
+        // Field size set up  2D array
```

Comment:
For methods, its better to use Summaries. Just type /// before the method and it should autogenerate the necessary tags.
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags

Reply:
Dont want to leave comment at each method, but general idea is to have these summaries for all public methods in your classes.